examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -93,7 +93,7 @@
 # Plot source space activity:
 
 # take absolute values for plotting
-stc.data[:, :] = np.abs(stc.data)
+np.abs(stc.data, out=stc.data)
```

Comment:
again, unrelated change ...

Reply:
I can live with it. I am getting old and more permissive... :)